ptbcontrib/persistent_jobstore/jobstore.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+#
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020-2021
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This file contains AdaptedSQLAlchemyJobStore."""
+
+from typing import Any
+import pickle
+import telegram
+import apscheduler.job
+
+from apscheduler.jobstores.sqlalchemy import SQLAlchemyJobStore
+from telegram.ext import CallbackContext, Dispatcher
+
+
+class AdaptedSQLAlchemyJobStore(SQLAlchemyJobStore):
+    """
+    Wraps apscheduler.SQLAlchemyJobStore to make telegram Job class storable.
+    """
+
+    def __init__(self, dispatcher: Dispatcher, *args: Any, **kwargs: Any) -> None:
+        """
+        :param dispatcher (:class:`telegram.ext.Dispatcher`): Dispatcher instance
```

Comment:
I know that we don't render the docs for ptbcontrib, but IMHO it would still be nice to stick to the google style docstrings